diff -rupN fluidsynth-1.1.5.old/doc/Doxyfile fluidsynth-1.1.5/doc/Doxyfile
--- fluidsynth-1.1.5.old/doc/Doxyfile	2011-09-04 03:38:58.000000000 -0400
+++ fluidsynth-1.1.5/doc/Doxyfile	2012-06-12 20:35:34.345619002 -0400
@@ -134,7 +134,7 @@ GENERATE_HTML          = YES
 HTML_OUTPUT            = html
 HTML_FILE_EXTENSION    = .html
 HTML_HEADER            = 
-HTML_FOOTER            = 
+HTML_FOOTER            = no_date_footer.html
 HTML_STYLESHEET        = 
 HTML_ALIGN_MEMBERS     = YES
 GENERATE_HTMLHELP      = NO
diff -rupN fluidsynth-1.1.5.old/doc/Doxyfile.cmake fluidsynth-1.1.5/doc/Doxyfile.cmake
--- fluidsynth-1.1.5.old/doc/Doxyfile.cmake	2011-09-04 03:38:58.000000000 -0400
+++ fluidsynth-1.1.5/doc/Doxyfile.cmake	2012-06-12 20:55:24.288334928 -0400
@@ -121,7 +121,7 @@ GENERATE_HTML = YES
 HTML_OUTPUT = html
 HTML_FILE_EXTENSION = .html
 HTML_HEADER = 
-HTML_FOOTER = 
+HTML_FOOTER = @CMAKE_SOURCE_DIR@/doc/no_date_footer.html
 HTML_STYLESHEET = 
 HTML_ALIGN_MEMBERS = YES
 GENERATE_HTMLHELP = NO
diff -rupN fluidsynth-1.1.5.old/doc/no_date_footer.html fluidsynth-1.1.5/doc/no_date_footer.html
--- fluidsynth-1.1.5.old/doc/no_date_footer.html	1969-12-31 19:00:00.000000000 -0500
+++ fluidsynth-1.1.5/doc/no_date_footer.html	2012-06-12 20:36:05.628479193 -0400
@@ -0,0 +1,6 @@
+<hr size="1"><address style="text-align: right;"><small>
+Generated for $projectname by <a href="http://www.doxygen.org/
+index.html"><img src="doxygen.png" alt="doxygen" align="middle" border="0"></a>
+$doxygenversion</small></address>
+</body>
+</html>
